docs/concepts/glossary.md

```diff
@@ -0,0 +1,166 @@
+Taskmanagers processes.
+
+#### Event
+
+An event is a statement about a change of the state of the domain modelled by the
+application. Events can be input and/or output of a stream or batch processing application.
+Events are special types of [records](./glossary#Record)
+
+#### ExecutionGraph
+
+see [Physical Graph](./glossary#physical-graph)
+
+#### Function
+
+Functions, or user-defined functions (UDFs), are implemented by the user and encapsulate the
+application logic of a Flink program. Most Functions are wrapped by a corresponding
+[Operator](./glossary#operator).
+
+#### Instance
+
+The term *instance* is used to describe a specific instance of a specific type (usually
+[Operator](./glossary#operator) or [Function](./glossary#function)) during runtime. As Apache Flink
+is mostly written in Java, this corresponds to the definition of *Instance* or *Object* in Java.
+In the context of Apache Flink, the term *parallel instance* is also frequently used to emphasize
+that multiple instances of the same [Operator](./glossary#operator) or
+[Function](./glossary#function) type are running in parallel.
+
+#### Flink Job
+
+A Flink Job is the runtime representation of a Flink program. A Flink Job can either be submitted
+to a long running [Flink Session Cluster](./glossary#flink-session-cluster) or it can be started as a
+self-contained [Flink Application Cluster](./glossary#flink-application-cluster).
+
+#### JobGraph
+
+see [Logical Graph](./glossary#logical-graph)
+
+#### Flink JobManager
+
+JobManagers are one of the components running in the
+[Flink JobManger Process](./glossary#flink-jobmanager-process). A JobManager is responsible for
+supervising the execution of the [Tasks](./glossary#task) of a single job.
+
+#### Logical Graph
+
+A logical graph is a directed graph describing the high-level logic of a stream processing program.
+The nodes are [Operators](./glossary#operator) and the edges indicate input/output-relationships or
+data streams or data sets.
+
+#### Managed State
+
+Managed State describes application state which has been registered with the framework. For
+Managed State, Apache Flink will take care about persistence and rescaling among other things.
+
+#### Flink JobManager Process
+
+The Job Manager Process is the master of a [Flink Cluster](./glossary#flink-cluster). It is called
+*JobManager* for historical reasons, but actually has actually contains three distinct components:
+Flink Resource Manager, Flink Dispatcher and one [Flink JobManager](./glossary#flink-jobmanager)
+per running [Flink Job](./glossary#flink-job).
+
+#### Operator
+
+Node of a [Logical Graph](./glossary#logical-graph). An Operator performs a certain operation,
+which is usually executed by a [Function](./glossary#function). Sources and Sinks are special
+Operators for data ingestion and data egress.
+
+#### Operator Chain
+
+An Operator Chain consists of one or more consecutive [Operators](./glossary#operator) without any
```

Comment:
I would not call a single operator an operator chain. IMO a chain consist of at least two operators.

Reply:
Ok, personally I think a chain of size 1 is perfectly fine, but I'll change it ;)